Makefile

```diff
@@ -211,5 +223,6 @@ distclean: clean
 	$(RM) *.zip
 	$(RM) -r $(OUT)/mini-gdbstub
 	-$(RM) $(OUT)/.config
+	$(MAKE) -C src/SoftFloat-3e/build/Linux-RISCV-GCC/ clean
```

Comment:
It should begin with a `-` character, indicating that it is acceptable to fail.
See https://www.gnu.org/software/make/manual/html_node/Errors.html